2025-02-08 01:43:56,473 - LoudVA - INFO - [LoudController] - Starting LoudController...
2025-02-08 01:43:56,473 - LoudVA - INFO - [LoudController] - Press CTRL+C to stop the controller.
2025-02-08 01:43:56,473 - LoudVA - INFO - [LoudController] - Debug mode: False
2025-02-08 01:43:56,495 - LoudVA - INFO - [LoudController] - Selected scheduler: loud
2025-02-08 01:43:56,498 - LoudVA - INFO - [LoudServer] - Starting LoudVA server...
 * Serving Flask app 'LoudServer'
 * Debug mode: off
WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
 * Running on http://127.0.0.1:5000
Press CTRL+C to quit
2025-02-08 01:43:58,453 - LoudVA - INFO - [DeviceData] - Training the LoudCostPredictor...
2025-02-08 01:43:58,628 - LoudVA - INFO - [DeviceData] - LoudCostPredictor training complete. Training time: 0.17539095878601074 seconds.
2025-02-08 01:43:58,629 - LoudVA - INFO - [DeviceData] - Using profiling data for decision making.
2025-02-08 01:43:58,629 - LoudVA - INFO - [DeviceData] - Filling missing profile data with predictions.
2025-02-08 01:43:58,644 - LoudVA - INFO - [DeviceData] - Loaded profile from /home/iloudaros/Desktop/LoudVA/LoudController/../measurements/archive/Representative/Profiling.csv
2025-02-08 01:43:58,644 - LoudVA - INFO - [DeviceData] - Loaded profile from /home/iloudaros/Desktop/LoudVA/LoudController/../measurements/archive/Representative/xavier-nx-00/measurements/xavier-nx-00_filtered_freqs.csv
2025-02-08 01:43:58,645 - LoudVA - INFO - [DeviceData] - Loaded profile from /home/iloudaros/Desktop/LoudVA/LoudController/../measurements/archive/Representative/LoudJetson0/measurements/LoudJetson0_filtered_freqs.csv
2025-02-08 01:43:58,645 - LoudVA - INFO - [DeviceData] - Loaded GPU specs from /home/iloudaros/Desktop/LoudVA/LoudController/../data/devices/gpu_specs.csv
2025-02-08 01:43:58,675 - LoudVA - INFO - [DeviceData] - Initial frequency on agx-xavier-00: 114750000
2025-02-08 01:43:58,676 - LoudVA - INFO - [DeviceData] - agx-xavier-00 : Filling missing profile data...
2025-02-08 01:43:58,676 - LoudVA - INFO - [DeviceData] - agx-xavier-00 : Missing profile data filled with predictions.
2025-02-08 01:43:58,676 - LoudVA - INFO - [DeviceData] - Devices initialized successfully
2025-02-08 01:43:58,690 - LoudVA - INFO - [LoudScheduler] - Starting LoudScheduler
2025-02-08 01:44:02,753 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
127.0.0.1 - - [08/Feb/2025 01:44:58] "GET / HTTP/1.1" 200 -
2025-02-08 01:44:59,931 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:44:59,946 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:04,894 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 7.0
2025-02-08 01:45:07,027 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 39
2025-02-08 01:45:07,096 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:45:10,019 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:45:10,051 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:12,378 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: d26ea8f3-b7f5-41b0-8ce0-50e49aab52fa
127.0.0.1 - - [08/Feb/2025 01:45:12] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:12,381 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: b05a1f7c-defb-4554-9a93-be9f3f8cdf0e
127.0.0.1 - - [08/Feb/2025 01:45:12] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:12,888 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
2025-02-08 01:45:14,936 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:45:15,078 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:15,089 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:15,145 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:45:16,840 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:19,953 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:45:20,968 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
2025-02-08 01:45:24,282 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 1032750000, batch size 40
2025-02-08 01:45:24,292 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:24,352 - LoudVA - INFO - [DeviceData] - Frequency set to 1032750000 MHz on agx-xavier-00
2025-02-08 01:45:24,653 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 06fb2127-6115-42ab-8a88-b57ff2886b87
127.0.0.1 - - [08/Feb/2025 01:45:24] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:24,904 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 12.0
2025-02-08 01:45:25,848 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:29,924 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f45649a6-29ab-44d9-ba0a-d1d1cf098a68
127.0.0.1 - - [08/Feb/2025 01:45:29] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:29,990 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 17
2025-02-08 01:45:30,000 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:30,007 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:45:30,059 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:45:31,339 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:32,527 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 9d64f3d4-894d-4414-a737-2d46b69e157b
127.0.0.1 - - [08/Feb/2025 01:45:32] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:33,403 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:34,985 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:45:35,003 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:37,974 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 11.0
2025-02-08 01:45:39,028 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
2025-02-08 01:45:39,952 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:45:40,039 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:40,050 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:40,108 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:45:40,837 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: d402c77c-9888-4503-890c-fbcc4f72babe
127.0.0.1 - - [08/Feb/2025 01:45:40] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:40,846 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 92357d46-d81e-437d-9e0e-133fa241b1bd
127.0.0.1 - - [08/Feb/2025 01:45:40] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:41,804 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:42,976 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:45:44,922 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:45:44,942 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:44,953 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:46,215 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:46,617 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 6e20c748-76c2-4a95-8f09-2625f952ebf6
127.0.0.1 - - [08/Feb/2025 01:45:46] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:47,970 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:45:49,982 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 21.0
2025-02-08 01:45:50,015 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:50,029 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:51,238 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:51,963 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: a6ce4e76-0cc4-429f-ac83-edd87c5bf4dc
127.0.0.1 - - [08/Feb/2025 01:45:51] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:51,972 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: d43f87fb-f313-48e7-aecf-8e9d47c14570
127.0.0.1 - - [08/Feb/2025 01:45:51] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:53,007 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:45:53,037 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:53,047 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:54,233 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:54,918 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:45:58,044 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:45:58,161 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:45:58,175 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:45:59,357 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:45:59,447 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: bfbb633b-fea6-4d25-80d8-e560f3fc57ef
127.0.0.1 - - [08/Feb/2025 01:45:59] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:59,454 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 345246fc-e569-4bb4-a18d-0911a74de7e4
127.0.0.1 - - [08/Feb/2025 01:45:59] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:59,470 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2984e16b-bc54-4875-b235-1fc4503a74f5
127.0.0.1 - - [08/Feb/2025 01:45:59] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:45:59,911 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:46:02,332 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 3b96bc20-60ba-40ab-aa39-6e33946b7162
127.0.0.1 - - [08/Feb/2025 01:46:02] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:02,975 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 10.0
2025-02-08 01:46:03,003 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:03,015 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:04,201 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:04,924 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:46:07,357 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 7d530fcd-9e42-4fd9-86a3-78661167ae18
127.0.0.1 - - [08/Feb/2025 01:46:07] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:08,017 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 22.0
2025-02-08 01:46:08,198 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:08,209 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:09,395 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:09,951 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:46:10,176 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:10,186 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:11,087 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 6c708a67-a8c5-4026-8581-d316966ae330
127.0.0.1 - - [08/Feb/2025 01:46:11] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:11,115 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f2af4b39-0a18-4212-bb47-372206f30a3b
127.0.0.1 - - [08/Feb/2025 01:46:11] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:11,383 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:12,980 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:46:14,926 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:46:17,138 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
2025-02-08 01:46:18,035 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:46:18,126 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:18,137 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:18,231 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:46:19,153 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: d6ad2859-75e5-45c4-8ef6-317e1750bb09
127.0.0.1 - - [08/Feb/2025 01:46:19] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:19,158 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 1e53d7ff-d38a-41d7-8ece-037a0f495d34
127.0.0.1 - - [08/Feb/2025 01:46:19] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:19,172 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 151ea94a-f335-4a67-bfd6-31fbaa459adb
127.0.0.1 - - [08/Feb/2025 01:46:19] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:19,884 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:19,896 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:46:20,404 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: b17dd219-a220-4b9b-b5c2-e3493e3b7baa
127.0.0.1 - - [08/Feb/2025 01:46:20] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:22,956 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:46:24,911 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:46:24,975 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:24,984 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 40fccb39-46c6-4832-bcdd-4d15ab07c4de
127.0.0.1 - - [08/Feb/2025 01:46:24] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:24,986 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:24,991 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: b4cb9d52-e428-4679-8f5d-143c598ea95c
127.0.0.1 - - [08/Feb/2025 01:46:24] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:26,169 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:28,032 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:46:28,247 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:28,260 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:29,448 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:29,910 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:46:31,522 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 1f4d4ad5-70d8-43f6-a59f-b2286a38677a
127.0.0.1 - - [08/Feb/2025 01:46:31] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:31,525 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f7e244b8-9c70-464a-b87f-9a9b41d21ab9
127.0.0.1 - - [08/Feb/2025 01:46:31] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:31,535 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 5d5cbff0-ba1b-4e69-99ce-9a1663989e87
127.0.0.1 - - [08/Feb/2025 01:46:31] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:32,993 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:46:33,124 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:33,137 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:34,335 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:34,969 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:46:35,249 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:35,259 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:36,471 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:36,633 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 1ffbf7d6-1bb8-49f8-8b6d-973bbb76a849
127.0.0.1 - - [08/Feb/2025 01:46:36] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:36,644 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0a096fef-a442-471e-a01a-ca96647dab2c
127.0.0.1 - - [08/Feb/2025 01:46:36] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:37,993 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:46:39,914 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:46:39,942 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:39,952 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:41,136 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:43,110 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:46:43,182 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:43,192 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:43,795 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: da097c1b-1645-4231-b077-d4abbd03fa00
127.0.0.1 - - [08/Feb/2025 01:46:43] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:43,839 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 01ee30d9-bfcf-436a-8fe6-e6cf9947127e
127.0.0.1 - - [08/Feb/2025 01:46:43] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:44,377 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:44,962 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:46:45,288 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:45,298 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:46,503 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:48,003 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:46:48,258 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: cfe145fb-a8ea-45cc-ac56-044f3ba3d51f
127.0.0.1 - - [08/Feb/2025 01:46:48] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:46:48,398 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:48,408 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:49,595 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:50,009 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:46:51,981 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:46:52,003 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:52,022 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:53,024 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:46:53,205 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:54,928 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:46:54,998 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:55,035 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:55,597 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:46:56,359 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:46:57,094 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 20.0
2025-02-08 01:46:57,434 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:46:57,444 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:46:58,047 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:46:58,636 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:00,053 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:47:00,141 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:00,142 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: aa0dc2e0-bf0e-47f8-821d-ed2176a0169a
127.0.0.1 - - [08/Feb/2025 01:47:00] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:00,152 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:00,155 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 4f9cc8c6-8d23-44a9-8901-f07013ada189
127.0.0.1 - - [08/Feb/2025 01:47:00] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:00,652 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 12.0
2025-02-08 01:47:01,345 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:01,355 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:01,964 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:02,544 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:02,968 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 11.0
2025-02-08 01:47:05,091 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 20.0
2025-02-08 01:47:05,335 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:05,346 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:06,612 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:07,156 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:47:07,508 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:07,526 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:08,228 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:47:08,894 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:08,904 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:10,096 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:10,131 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 22.0
2025-02-08 01:47:10,746 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:10,757 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:11,941 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:12,288 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:47:12,744 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:12,755 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:12,798 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0598f5b8-a23e-4ccc-8192-7517f13dcde7
127.0.0.1 - - [08/Feb/2025 01:47:12] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:12,988 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 11.0
2025-02-08 01:47:13,948 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:15,070 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:47:15,511 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:15,522 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:16,763 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:16,942 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:47:17,979 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:18,977 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:47:19,310 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:19,321 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:20,188 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:20,547 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:20,923 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:20,933 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:22,267 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:22,473 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:47:23,283 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:47:23,322 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:23,337 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:23,823 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:24,527 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:24,537 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:25,160 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:47:25,732 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:27,390 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 20.0
2025-02-08 01:47:27,518 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:27,529 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:28,623 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 21.0
2025-02-08 01:47:28,718 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:28,750 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:28,761 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:28,925 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:47:29,951 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:29,961 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:30,286 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:31,151 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:31,161 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:31,673 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 84589160-3607-40c0-be76-db5c061077c5
127.0.0.1 - - [08/Feb/2025 01:47:31] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:31,786 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 83f0ec16-90d8-4e2e-b676-c985cc931e76
127.0.0.1 - - [08/Feb/2025 01:47:31] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:32,125 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:47:32,349 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:33,225 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:47:33,685 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:33,696 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:33,876 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:47:34,885 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:35,005 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:47:37,614 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:47:37,712 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:37,723 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:38,370 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:47:38,975 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:38,985 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:39,084 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:47:40,170 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:40,180 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:41,367 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:42,158 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:47:42,636 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:42,647 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:43,235 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:47:43,567 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 2.0
2025-02-08 01:47:43,930 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:43,959 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:47:43,961 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:44,520 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 4.0
2025-02-08 01:47:45,129 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 1032750000, batch size 40
2025-02-08 01:47:45,140 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:45,254 - LoudVA - INFO - [DeviceData] - Frequency set to 1032750000 MHz on agx-xavier-00
2025-02-08 01:47:45,450 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 12.0
2025-02-08 01:47:46,518 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:47:46,686 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:46,831 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 38f92dbb-88f0-4a77-a3c7-811645821658
127.0.0.1 - - [08/Feb/2025 01:47:46] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:47,382 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:47:47,473 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:47:48,597 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 11.0
2025-02-08 01:47:49,049 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:47:49,090 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:47:49,210 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:49,221 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:49,343 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:47:49,710 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 11.0
2025-02-08 01:47:50,080 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 4717dd32-357c-490e-90f8-71091403db3c
127.0.0.1 - - [08/Feb/2025 01:47:50] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:50,246 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: ce1775e8-c2f9-4051-8ac8-9b5b80d51d9f
127.0.0.1 - - [08/Feb/2025 01:47:50] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:50,698 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:50,977 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:50,987 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:51,597 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 4.0
2025-02-08 01:47:52,176 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:52,535 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:47:52,738 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 31
2025-02-08 01:47:52,748 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:52,874 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:47:53,469 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:47:53,706 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:53,783 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:47:53,920 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:53,932 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:54,077 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:55,122 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:55,132 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:47:55,450 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 28734075-d79c-4b52-8d96-eec8fc8c520d
127.0.0.1 - - [08/Feb/2025 01:47:55] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:55,537 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c8e0f021-cc6c-4868-a7a9-fe62aa6885a8
127.0.0.1 - - [08/Feb/2025 01:47:55] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:55,658 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: adb5892b-7113-45c7-a4d7-0555e66aa0f5
127.0.0.1 - - [08/Feb/2025 01:47:55] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:47:56,323 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:47:56,992 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 11.0
2025-02-08 01:47:58,268 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:47:59,227 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:47:59,328 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:47:59,341 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:00,523 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-51:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:02,091 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:48:02,992 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 8.0
Exception in thread Thread-53:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:04,378 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 20.0
2025-02-08 01:48:04,728 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:04,740 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:05,926 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-55:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:07,508 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:48:08,293 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 6.0
2025-02-08 01:48:08,363 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:08,374 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:09,396 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:48:09,566 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:48:09,649 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 1032750000, batch size 40
2025-02-08 01:48:09,660 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-57:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:09,772 - LoudVA - INFO - [DeviceData] - Frequency set to 1032750000 MHz on agx-xavier-00
Exception in thread Thread-59:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:11,212 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-61:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:12,944 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:48:13,491 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:13,502 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:13,839 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:48:13,911 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:48:14,592 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
Exception in thread Thread-63:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:15,254 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:15,264 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:16,456 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-65:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:17,476 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 22.0
2025-02-08 01:48:17,988 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:17,999 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:18,873 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:48:18,926 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:48:19,185 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:48:19,326 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:19,351 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:20,679 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-67:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-69:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:23,197 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 20.0
2025-02-08 01:48:23,747 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:23,759 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:24,050 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 22.0
2025-02-08 01:48:24,459 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:48:24,949 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:24,961 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-71:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:26,244 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-73:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:27,692 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:48:27,998 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:28,009 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:28,877 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:48:29,147 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:48:29,197 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:48:29,456 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:29,467 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-75:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:30,861 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-77:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-79:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:32,420 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:48:32,652 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:32,663 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-81:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:33,578 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
2025-02-08 01:48:33,857 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:48:34,010 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:48:34,162 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:34,173 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:35,362 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-83:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:37,809 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
2025-02-08 01:48:38,213 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:38,223 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:39,143 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
2025-02-08 01:48:39,280 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
Exception in thread Thread-85:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:39,411 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:48:39,955 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:39,966 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-87:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:41,150 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-89:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:43,139 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:48:43,530 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:43,541 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:43,797 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 20.0
Exception in thread Thread-91:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:44,558 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:48:44,956 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:44,968 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-93:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:46,153 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-95:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:47,692 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
2025-02-08 01:48:47,854 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:47,864 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:49,047 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:48:49,387 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 19.0
Exception in thread Thread-97:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:53,138 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:48:53,412 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:48:53,424 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-99:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:54,016 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 14.0
Exception in thread Thread-101:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
2025-02-08 01:48:54,802 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:54,813 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:48:56,003 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-103:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:57,157 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 13.0
Exception in thread Thread-105:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:48:59,411 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:49:00,139 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
Exception in thread Thread-107:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:02,815 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:49:03,128 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:03,139 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:03,252 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:49:04,880 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:49:04,923 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:49:05,437 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:05,449 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:06,637 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-109:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:07,799 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:49:08,419 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:08,430 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:08,902 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:49:09,620 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-111:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-113:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:13,695 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 18.0
2025-02-08 01:49:14,176 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
2025-02-08 01:49:14,379 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:14,390 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:14,501 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
2025-02-08 01:49:14,716 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 17.0
Exception in thread Thread-115:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:16,144 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:49:17,762 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:49:17,896 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:17,907 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:18,808 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 9.0
2025-02-08 01:49:19,102 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-117:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-119:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-121:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:23,902 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 31
2025-02-08 01:49:23,912 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:24,870 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 1032750000, batch size 2
2025-02-08 01:49:24,997 - LoudVA - INFO - [DeviceData] - Frequency set to 1032750000 MHz on agx-xavier-00
2025-02-08 01:49:25,286 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:49:25,380 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:25,994 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-123:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-125:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-127:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:29,837 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:49:30,082 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:30,093 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:30,205 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
Exception in thread Thread-129:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:31,907 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-131:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:35,234 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
2025-02-08 01:49:35,593 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:35,604 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-133:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:36,792 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:49:39,483 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
2025-02-08 01:49:39,845 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:39,855 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
Exception in thread Thread-135:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:41,041 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-137:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:44,846 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 14
Exception in thread Thread-139:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:46,282 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 21.0
2025-02-08 01:49:46,534 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
Exception in thread Thread-141:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:49,391 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 15.0
Exception in thread Thread-143:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:49,620 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:49,630 - LoudVA - WARNING - [LoudScheduler] - No available devices found. Returning None. Waiting for devices to become available.
2025-02-08 01:49:50,816 - LoudVA - INFO - [LoudScheduler] - Latency constraint allows for waiting, holding for more images.
2025-02-08 01:49:54,644 - LoudVA - INFO - [LoudServer] - Received inference request with latency constraint: 16.0
Exception in thread Thread-145:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:56,337 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
Exception in thread Thread-147:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:49:59,156 - LoudVA - INFO - [LoudScheduler] - Dispatching to agx-xavier-00 with frequency 905250000, batch size 51
2025-02-08 01:49:59,275 - LoudVA - INFO - [DeviceData] - Frequency set to 905250000 MHz on agx-xavier-00
Exception in thread Thread-149:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:50:05,286 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f7e8bb41-2af4-468f-b6d0-4ad11a1db37b
127.0.0.1 - - [08/Feb/2025 01:50:05] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:06,255 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c1e6f64d-5405-4e1d-9ea8-6c13211fdeaf
127.0.0.1 - - [08/Feb/2025 01:50:06] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:06,365 - LoudVA - INFO - [DeviceData] - Frequency set to 114750000 MHz on agx-xavier-00
Exception in thread Thread-151:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
Exception in thread Thread-153:
Traceback (most recent call last):
  File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.8/threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/LoudScheduler.py", line 249, in dispatch_request
    [response] = device.inference(images, batch_size)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/DeviceData.py", line 258, in inference
    response = triton_client.inference(**args)
  File "/home/iloudaros/Desktop/LoudVA/LoudController/triton_client.py", line 337, in inference
    triton_client.infer(model_name,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 1469, in infer
    response = self._post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/tritonclient/http/_client.py", line 284, in _post
    response = self._client_stub.post(request_uri=request_uri,
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 272, in post
    return self.request(METHOD_POST, request_uri, body=body, headers=headers)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 250, in request
    raise e
  File "/home/iloudaros/.local/lib/python3.8/site-packages/geventhttpclient/client.py", line 231, in request
    sock.sendall(_request + body)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 702, in sendall
    return _sendall(self, data_memory, flags)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 391, in _sendall
    timeleft = __send_chunk(socket, chunk, flags, timeleft, end)
  File "/home/iloudaros/.local/lib/python3.8/site-packages/gevent/_socketcommon.py", line 328, in __send_chunk
    raise timeout('timed out')
socket.timeout: timed out
2025-02-08 01:50:21,916 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0de87749-f570-434d-9593-a75705d1b8f0
127.0.0.1 - - [08/Feb/2025 01:50:21] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:25,397 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 23045d3b-6827-45e0-b97b-861850172eae
127.0.0.1 - - [08/Feb/2025 01:50:25] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:27,088 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: a32c0a68-f428-4755-aa69-40ca5fae04be
127.0.0.1 - - [08/Feb/2025 01:50:27] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:32,788 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c41cf79b-7611-45e5-acc6-17354d617206
127.0.0.1 - - [08/Feb/2025 01:50:32] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:37,937 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 6adca1eb-c6e8-4a3f-96fa-25d5fc7b1ac9
127.0.0.1 - - [08/Feb/2025 01:50:37] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:47,066 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: fd7444da-116c-4e26-99f6-6d2d4ea0e162
127.0.0.1 - - [08/Feb/2025 01:50:47] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:48,500 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: dea5b6f3-bc6f-4578-b183-6d0ae53d35f0
127.0.0.1 - - [08/Feb/2025 01:50:48] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:52,453 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 640b7f72-4f8c-4bef-95e6-1e01cface374
127.0.0.1 - - [08/Feb/2025 01:50:52] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:52,934 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: ca7c45ca-5380-4ff4-aeaf-220d275e24b7
127.0.0.1 - - [08/Feb/2025 01:50:52] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:50:53,040 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 072d367d-6a9e-4a12-ac6f-d0e2e8938cc2
127.0.0.1 - - [08/Feb/2025 01:50:53] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:51:57,233 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 492868fe-ccfb-44a3-9311-c4224085a778
2025-02-08 01:51:57,930 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 492868fe-ccfb-44a3-9311-c4224085a778
127.0.0.1 - - [08/Feb/2025 01:51:57] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:51:58,131 - LoudVA - WARNING - [LoudServer] - Timeout reached for request ec2f59a6-8ab2-4615-ae7a-a95be8188637
2025-02-08 01:51:58,786 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: ec2f59a6-8ab2-4615-ae7a-a95be8188637
127.0.0.1 - - [08/Feb/2025 01:51:58] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:00,321 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 409cd82a-ded8-4300-978d-5b41df8225d8
2025-02-08 01:52:00,765 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 66e72c23-91a3-4f4f-a912-5baef80e64da
2025-02-08 01:52:01,152 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 66e72c23-91a3-4f4f-a912-5baef80e64da
127.0.0.1 - - [08/Feb/2025 01:52:01] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:01,174 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 409cd82a-ded8-4300-978d-5b41df8225d8
127.0.0.1 - - [08/Feb/2025 01:52:01] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:02,008 - LoudVA - WARNING - [LoudServer] - Timeout reached for request d825cadf-64a8-43d8-aead-32ae4085ab22
2025-02-08 01:52:02,195 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: d825cadf-64a8-43d8-aead-32ae4085ab22
127.0.0.1 - - [08/Feb/2025 01:52:02] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:03,010 - LoudVA - WARNING - [LoudServer] - Timeout reached for request c631d65e-c87f-431b-9266-76fda139757e
2025-02-08 01:52:03,079 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c631d65e-c87f-431b-9266-76fda139757e
127.0.0.1 - - [08/Feb/2025 01:52:03] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:05,172 - LoudVA - WARNING - [LoudServer] - Timeout reached for request ece3c4d3-4baf-41b4-9090-42503453a129
2025-02-08 01:52:05,695 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: ece3c4d3-4baf-41b4-9090-42503453a129
127.0.0.1 - - [08/Feb/2025 01:52:05] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:07,365 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 06cee993-cc6f-4e9b-a7a4-f2beaf0806f7
2025-02-08 01:52:08,101 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 06cee993-cc6f-4e9b-a7a4-f2beaf0806f7
127.0.0.1 - - [08/Feb/2025 01:52:08] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:08,258 - LoudVA - WARNING - [LoudServer] - Timeout reached for request e250838b-f22e-44dc-9dda-1772f0b2dc6d
2025-02-08 01:52:08,924 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: e250838b-f22e-44dc-9dda-1772f0b2dc6d
127.0.0.1 - - [08/Feb/2025 01:52:08] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:10,445 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 043df41a-ca56-4592-8f7a-4476815cf0a5
2025-02-08 01:52:11,240 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 043df41a-ca56-4592-8f7a-4476815cf0a5
127.0.0.1 - - [08/Feb/2025 01:52:11] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:12,391 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 04eab0bd-5254-4ef2-8e7b-d28bc3028341
2025-02-08 01:52:13,022 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 0b0f1b5f-ed79-493b-a1c8-2e63d438da9c
2025-02-08 01:52:13,119 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0b0f1b5f-ed79-493b-a1c8-2e63d438da9c
127.0.0.1 - - [08/Feb/2025 01:52:13] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:13,133 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 04eab0bd-5254-4ef2-8e7b-d28bc3028341
127.0.0.1 - - [08/Feb/2025 01:52:13] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:15,255 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f705a95d-4ccf-4f1d-8bbd-34ec7c7d3863
2025-02-08 01:52:15,918 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f705a95d-4ccf-4f1d-8bbd-34ec7c7d3863
127.0.0.1 - - [08/Feb/2025 01:52:15] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:17,007 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 9a0d1f7d-8f02-4ec8-ac0a-665166c1de1d
2025-02-08 01:52:17,184 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 9a0d1f7d-8f02-4ec8-ac0a-665166c1de1d
127.0.0.1 - - [08/Feb/2025 01:52:17] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:18,027 - LoudVA - WARNING - [LoudServer] - Timeout reached for request b51ff5de-38f2-4f8b-83ab-8d114bb3409d
2025-02-08 01:52:18,148 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: b51ff5de-38f2-4f8b-83ab-8d114bb3409d
127.0.0.1 - - [08/Feb/2025 01:52:18] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:19,045 - LoudVA - WARNING - [LoudServer] - Timeout reached for request a37e4c8c-3003-42e6-8881-bf2591223edc
2025-02-08 01:52:19,457 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: a37e4c8c-3003-42e6-8881-bf2591223edc
127.0.0.1 - - [08/Feb/2025 01:52:19] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:20,557 - LoudVA - WARNING - [LoudServer] - Timeout reached for request fb977853-a9fb-4934-addf-416f311b523b
2025-02-08 01:52:21,473 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: fb977853-a9fb-4934-addf-416f311b523b
127.0.0.1 - - [08/Feb/2025 01:52:21] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:22,700 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 17cb99a6-8da8-4699-9a07-680c7125ccd7
2025-02-08 01:52:23,316 - LoudVA - WARNING - [LoudServer] - Timeout reached for request c637a012-7d64-44c8-9d26-979e9cb833f9
2025-02-08 01:52:23,406 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 17cb99a6-8da8-4699-9a07-680c7125ccd7
127.0.0.1 - - [08/Feb/2025 01:52:23] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:23,908 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c637a012-7d64-44c8-9d26-979e9cb833f9
2025-02-08 01:52:23,915 - LoudVA - WARNING - [LoudServer] - Timeout reached for request ae5bc8ed-bd76-49e6-b0fb-108a2e3e6935
127.0.0.1 - - [08/Feb/2025 01:52:23] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:24,121 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: ae5bc8ed-bd76-49e6-b0fb-108a2e3e6935
127.0.0.1 - - [08/Feb/2025 01:52:24] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:25,296 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 00dbcc04-6836-4bcb-be14-e6a326a5883a
2025-02-08 01:52:25,712 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 00dbcc04-6836-4bcb-be14-e6a326a5883a
127.0.0.1 - - [08/Feb/2025 01:52:25] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:27,650 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 11ae25a5-4983-4736-8343-acabe687cbea
2025-02-08 01:52:28,303 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 11ae25a5-4983-4736-8343-acabe687cbea
127.0.0.1 - - [08/Feb/2025 01:52:28] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:28,971 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f7e7c582-3668-441f-9d86-99f91e12495b
2025-02-08 01:52:29,040 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 505b2436-415c-4d7b-8f05-d725274326f9
2025-02-08 01:52:29,356 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 505b2436-415c-4d7b-8f05-d725274326f9
127.0.0.1 - - [08/Feb/2025 01:52:29] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:29,480 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f7e7c582-3668-441f-9d86-99f91e12495b
127.0.0.1 - - [08/Feb/2025 01:52:29] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:30,628 - LoudVA - WARNING - [LoudServer] - Timeout reached for request bf0e392c-3b5e-4b22-8bd7-a824339d694d
2025-02-08 01:52:31,063 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: bf0e392c-3b5e-4b22-8bd7-a824339d694d
127.0.0.1 - - [08/Feb/2025 01:52:31] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:32,311 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 1f43ec50-6c5b-4f6d-a23b-26e84b76cf22
2025-02-08 01:52:32,587 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 1f43ec50-6c5b-4f6d-a23b-26e84b76cf22
127.0.0.1 - - [08/Feb/2025 01:52:32] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:33,332 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 576affef-2331-474c-9f35-026a04b7fe07
2025-02-08 01:52:33,637 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 576affef-2331-474c-9f35-026a04b7fe07
127.0.0.1 - - [08/Feb/2025 01:52:33] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:33,904 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 26b22d09-3c7e-4e4d-bad9-e720dd0e880f
2025-02-08 01:52:34,050 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 26b22d09-3c7e-4e4d-bad9-e720dd0e880f
127.0.0.1 - - [08/Feb/2025 01:52:34] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:35,107 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f770bb14-6759-4965-9713-8af6d9a9f77d
2025-02-08 01:52:35,296 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f770bb14-6759-4965-9713-8af6d9a9f77d
127.0.0.1 - - [08/Feb/2025 01:52:35] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:37,841 - LoudVA - WARNING - [LoudServer] - Timeout reached for request cc9d44bc-f32d-45b4-8777-18bdf5b597cb
2025-02-08 01:52:38,410 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: cc9d44bc-f32d-45b4-8777-18bdf5b597cb
127.0.0.1 - - [08/Feb/2025 01:52:38] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:38,471 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 8d3d31e3-e07e-48d9-b18b-65d4b69fb8ad
2025-02-08 01:52:38,707 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 8d3d31e3-e07e-48d9-b18b-65d4b69fb8ad
127.0.0.1 - - [08/Feb/2025 01:52:38] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:39,141 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 62ba8b6e-975f-4147-980e-89f049c06503
2025-02-08 01:52:39,508 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 62ba8b6e-975f-4147-980e-89f049c06503
127.0.0.1 - - [08/Feb/2025 01:52:39] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:42,288 - LoudVA - WARNING - [LoudServer] - Timeout reached for request eda062ae-782f-42cd-a64f-2e0266eb0aec
2025-02-08 01:52:42,764 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: eda062ae-782f-42cd-a64f-2e0266eb0aec
127.0.0.1 - - [08/Feb/2025 01:52:42] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:43,356 - LoudVA - WARNING - [LoudServer] - Timeout reached for request c090ec71-f395-4157-a971-d3e9b4eab24c
2025-02-08 01:52:43,574 - LoudVA - WARNING - [LoudServer] - Timeout reached for request e860fcff-6f3e-4f48-9eb4-f75305b0f1ce
2025-02-08 01:52:43,590 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: e860fcff-6f3e-4f48-9eb4-f75305b0f1ce
127.0.0.1 - - [08/Feb/2025 01:52:43] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:43,897 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c090ec71-f395-4157-a971-d3e9b4eab24c
127.0.0.1 - - [08/Feb/2025 01:52:43] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:44,076 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 107550bd-02b1-4fc3-b70e-45885696169f
2025-02-08 01:52:44,223 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 107550bd-02b1-4fc3-b70e-45885696169f
127.0.0.1 - - [08/Feb/2025 01:52:44] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:44,556 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f6145468-ce90-4ef6-870a-ee845857fe78
2025-02-08 01:52:44,603 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f6145468-ce90-4ef6-870a-ee845857fe78
127.0.0.1 - - [08/Feb/2025 01:52:44] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:45,470 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f5881116-a2c8-46bc-a5e7-e31911f50b0e
2025-02-08 01:52:45,491 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f5881116-a2c8-46bc-a5e7-e31911f50b0e
127.0.0.1 - - [08/Feb/2025 01:52:45] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:46,536 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 0cc45bab-60fc-45b6-bafc-b5d82c5beff6
2025-02-08 01:52:46,568 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0cc45bab-60fc-45b6-bafc-b5d82c5beff6
127.0.0.1 - - [08/Feb/2025 01:52:46] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:47,434 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 71398e06-f5fa-4a24-aeb4-72b2ac3a75fc
2025-02-08 01:52:47,490 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 76bc35a6-df86-4275-8eb3-2c26f992f3a5
2025-02-08 01:52:47,495 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 76bc35a6-df86-4275-8eb3-2c26f992f3a5
127.0.0.1 - - [08/Feb/2025 01:52:47] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:47,515 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 71398e06-f5fa-4a24-aeb4-72b2ac3a75fc
127.0.0.1 - - [08/Feb/2025 01:52:47] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:48,621 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 3072243c-47f1-450a-961e-beb948b31804
2025-02-08 01:52:48,631 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 3072243c-47f1-450a-961e-beb948b31804
127.0.0.1 - - [08/Feb/2025 01:52:48] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:49,145 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 6e230832-a26a-407e-b5a2-3d88c0b81a73
2025-02-08 01:52:49,170 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 82d7d179-7bdf-43f3-9947-0023acef81ce
2025-02-08 01:52:49,221 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 6e230832-a26a-407e-b5a2-3d88c0b81a73
127.0.0.1 - - [08/Feb/2025 01:52:49] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:49,311 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 82d7d179-7bdf-43f3-9947-0023acef81ce
127.0.0.1 - - [08/Feb/2025 01:52:49] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:49,747 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 4b9e44ad-da63-4e3a-a402-a55acb60e119
2025-02-08 01:52:49,765 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 4b9e44ad-da63-4e3a-a402-a55acb60e119
127.0.0.1 - - [08/Feb/2025 01:52:49] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:50,731 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f4628e8d-142c-47e5-8c25-63f61ec60e42
2025-02-08 01:52:50,763 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f4628e8d-142c-47e5-8c25-63f61ec60e42
127.0.0.1 - - [08/Feb/2025 01:52:50] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:51,626 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 13b34d18-5202-4a07-82b6-043731ca5112
2025-02-08 01:52:51,646 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 13b34d18-5202-4a07-82b6-043731ca5112
127.0.0.1 - - [08/Feb/2025 01:52:51] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:52,560 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 07404f4e-15dc-445f-af44-039ec01bd191
2025-02-08 01:52:52,566 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 07404f4e-15dc-445f-af44-039ec01bd191
127.0.0.1 - - [08/Feb/2025 01:52:52] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:52,979 - LoudVA - WARNING - [LoudServer] - Timeout reached for request caa62e01-2384-41f9-a6c8-edfe2cccfb43
2025-02-08 01:52:53,116 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: caa62e01-2384-41f9-a6c8-edfe2cccfb43
127.0.0.1 - - [08/Feb/2025 01:52:53] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:53,482 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 2401f9b2-a7fa-4c47-86ef-f269f1a30e42
2025-02-08 01:52:53,489 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2401f9b2-a7fa-4c47-86ef-f269f1a30e42
127.0.0.1 - - [08/Feb/2025 01:52:53] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:53,877 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 2e9dd0f7-7a9d-47a9-a6c6-0d9bd0c95de1
2025-02-08 01:52:54,045 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2e9dd0f7-7a9d-47a9-a6c6-0d9bd0c95de1
127.0.0.1 - - [08/Feb/2025 01:52:54] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:54,145 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 2c5072f8-3822-450d-83a0-e3128bedae84
2025-02-08 01:52:54,190 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2c5072f8-3822-450d-83a0-e3128bedae84
127.0.0.1 - - [08/Feb/2025 01:52:54] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:57,007 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 413feaa5-4d45-450c-8cef-deed3d9bccd9
2025-02-08 01:52:57,039 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 413feaa5-4d45-450c-8cef-deed3d9bccd9
127.0.0.1 - - [08/Feb/2025 01:52:57] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:58,327 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 34026e2d-6e90-4dc4-905c-b2f2882498df
2025-02-08 01:52:58,382 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 34026e2d-6e90-4dc4-905c-b2f2882498df
127.0.0.1 - - [08/Feb/2025 01:52:58] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:52:59,249 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 6b47ce54-e495-4888-ba19-a717e7e8d914
2025-02-08 01:52:59,303 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 6b47ce54-e495-4888-ba19-a717e7e8d914
127.0.0.1 - - [08/Feb/2025 01:52:59] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:02,107 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 178b772b-c163-436b-9e9a-7e774a8cf275
2025-02-08 01:53:02,136 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 178b772b-c163-436b-9e9a-7e774a8cf275
127.0.0.1 - - [08/Feb/2025 01:53:02] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:03,012 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 1b0f337a-b9b8-45f8-abc9-48f3e0ed3b0a
2025-02-08 01:53:03,021 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 1b0f337a-b9b8-45f8-abc9-48f3e0ed3b0a
127.0.0.1 - - [08/Feb/2025 01:53:03] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:04,480 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 2af04aaf-8b09-4e2b-b196-a0de95ef8e8c
2025-02-08 01:53:04,556 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2af04aaf-8b09-4e2b-b196-a0de95ef8e8c
127.0.0.1 - - [08/Feb/2025 01:53:04] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:07,557 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 1b11412e-151a-434b-8567-055bffdd2f42
2025-02-08 01:53:07,608 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 1b11412e-151a-434b-8567-055bffdd2f42
127.0.0.1 - - [08/Feb/2025 01:53:07] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:08,327 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 0c5342ae-504a-4eed-875e-6d09c11a151d
2025-02-08 01:53:08,336 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0c5342ae-504a-4eed-875e-6d09c11a151d
127.0.0.1 - - [08/Feb/2025 01:53:08] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:09,522 - LoudVA - WARNING - [LoudServer] - Timeout reached for request c3a61551-e093-4c8f-81b8-f704aa3bf5b3
2025-02-08 01:53:09,606 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c3a61551-e093-4c8f-81b8-f704aa3bf5b3
127.0.0.1 - - [08/Feb/2025 01:53:09] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:13,031 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 3a5aad99-f835-41b2-9a95-3df3cc192f5a
2025-02-08 01:53:13,092 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 3a5aad99-f835-41b2-9a95-3df3cc192f5a
127.0.0.1 - - [08/Feb/2025 01:53:13] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:13,925 - LoudVA - WARNING - [LoudServer] - Timeout reached for request bf32c707-d551-42c2-a8d0-3f65cfdb357f
2025-02-08 01:53:13,933 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: bf32c707-d551-42c2-a8d0-3f65cfdb357f
127.0.0.1 - - [08/Feb/2025 01:53:13] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:14,667 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 152ecd17-f89d-48f0-96a7-39c62dbd57fd
2025-02-08 01:53:14,756 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 152ecd17-f89d-48f0-96a7-39c62dbd57fd
127.0.0.1 - - [08/Feb/2025 01:53:14] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:17,577 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 31e4b2ca-ae41-4c3c-bc23-53b13ddd89d5
2025-02-08 01:53:17,596 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 31e4b2ca-ae41-4c3c-bc23-53b13ddd89d5
127.0.0.1 - - [08/Feb/2025 01:53:17] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:18,948 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 4ecc22d2-3fcf-461f-ad20-98ad6bcb812c
2025-02-08 01:53:18,961 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 4ecc22d2-3fcf-461f-ad20-98ad6bcb812c
127.0.0.1 - - [08/Feb/2025 01:53:18] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:19,042 - LoudVA - WARNING - [LoudServer] - Timeout reached for request e942d2a5-abff-4c28-ba5f-7db94a793a8b
2025-02-08 01:53:19,085 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: e942d2a5-abff-4c28-ba5f-7db94a793a8b
127.0.0.1 - - [08/Feb/2025 01:53:19] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:23,319 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f3d206d5-c8b6-444a-ac95-f20d09b275ce
2025-02-08 01:53:23,345 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f3d206d5-c8b6-444a-ac95-f20d09b275ce
127.0.0.1 - - [08/Feb/2025 01:53:23] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:24,171 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 2f81ed11-8782-4ec0-9f58-e876d984a2c4
2025-02-08 01:53:24,204 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2f81ed11-8782-4ec0-9f58-e876d984a2c4
127.0.0.1 - - [08/Feb/2025 01:53:24] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:24,539 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 36f3f116-5d4b-4f24-97c1-226fe51d7b41
2025-02-08 01:53:24,548 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 36f3f116-5d4b-4f24-97c1-226fe51d7b41
127.0.0.1 - - [08/Feb/2025 01:53:24] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:27,764 - LoudVA - WARNING - [LoudServer] - Timeout reached for request a9b052d4-b82d-48e3-ba3a-23f60d335ea6
2025-02-08 01:53:27,808 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: a9b052d4-b82d-48e3-ba3a-23f60d335ea6
127.0.0.1 - - [08/Feb/2025 01:53:27] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:28,944 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 4816ede4-5e6d-4d53-a541-a4600b382ebf
2025-02-08 01:53:28,983 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 4816ede4-5e6d-4d53-a541-a4600b382ebf
127.0.0.1 - - [08/Feb/2025 01:53:28] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:29,182 - LoudVA - WARNING - [LoudServer] - Timeout reached for request e7b05cee-b470-4a85-bb88-d83311df0f82
2025-02-08 01:53:29,193 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: e7b05cee-b470-4a85-bb88-d83311df0f82
127.0.0.1 - - [08/Feb/2025 01:53:29] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:32,477 - LoudVA - WARNING - [LoudServer] - Timeout reached for request b91488bf-9105-4faf-8a64-7092aa37a737
2025-02-08 01:53:32,508 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: b91488bf-9105-4faf-8a64-7092aa37a737
127.0.0.1 - - [08/Feb/2025 01:53:32] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:33,631 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 895f6714-8ce2-4e89-954a-3bca9534c3ff
2025-02-08 01:53:33,654 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 895f6714-8ce2-4e89-954a-3bca9534c3ff
127.0.0.1 - - [08/Feb/2025 01:53:33] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:34,023 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 8fa96ba4-aae5-4877-b1c9-660295d4116a
2025-02-08 01:53:34,048 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 8fa96ba4-aae5-4877-b1c9-660295d4116a
127.0.0.1 - - [08/Feb/2025 01:53:34] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:37,864 - LoudVA - WARNING - [LoudServer] - Timeout reached for request ce28ddcb-fba5-4c8a-ab38-772d8e7eff89
2025-02-08 01:53:37,894 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: ce28ddcb-fba5-4c8a-ab38-772d8e7eff89
127.0.0.1 - - [08/Feb/2025 01:53:37] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:39,203 - LoudVA - WARNING - [LoudServer] - Timeout reached for request c947d7cc-543a-4957-b582-3f373cef54ba
2025-02-08 01:53:39,205 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: c947d7cc-543a-4957-b582-3f373cef54ba
127.0.0.1 - - [08/Feb/2025 01:53:39] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:39,401 - LoudVA - WARNING - [LoudServer] - Timeout reached for request be561d78-65fd-42b4-ae2c-08911a49c343
2025-02-08 01:53:39,436 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: be561d78-65fd-42b4-ae2c-08911a49c343
127.0.0.1 - - [08/Feb/2025 01:53:39] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:43,228 - LoudVA - WARNING - [LoudServer] - Timeout reached for request e8ea73fa-f14e-4c65-a2f0-fd8735e9b69e
2025-02-08 01:53:43,283 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: e8ea73fa-f14e-4c65-a2f0-fd8735e9b69e
127.0.0.1 - - [08/Feb/2025 01:53:43] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:43,842 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 5843e98f-2eec-45f2-af57-e162c0dca30b
2025-02-08 01:53:43,903 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 5843e98f-2eec-45f2-af57-e162c0dca30b
127.0.0.1 - - [08/Feb/2025 01:53:43] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:44,623 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 82b546f7-6802-4a4c-840e-d0c96ba6b3be
2025-02-08 01:53:44,642 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 82b546f7-6802-4a4c-840e-d0c96ba6b3be
127.0.0.1 - - [08/Feb/2025 01:53:44] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:47,763 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f8acb6ff-2472-4fe0-8e2e-6cfc43f989b4
2025-02-08 01:53:47,789 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f8acb6ff-2472-4fe0-8e2e-6cfc43f989b4
127.0.0.1 - - [08/Feb/2025 01:53:47] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:49,483 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 11fa2689-0963-4bbc-893d-a57c5a0e392a
2025-02-08 01:53:49,495 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 11fa2689-0963-4bbc-893d-a57c5a0e392a
127.0.0.1 - - [08/Feb/2025 01:53:49] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:53,269 - LoudVA - WARNING - [LoudServer] - Timeout reached for request f1b1c313-0623-42e8-8b85-af95b61dd8dc
2025-02-08 01:53:53,302 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: f1b1c313-0623-42e8-8b85-af95b61dd8dc
127.0.0.1 - - [08/Feb/2025 01:53:53] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:54,035 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 2ddf8d83-f2c7-40f1-8231-fc410dbffcce
2025-02-08 01:53:54,042 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 2ddf8d83-f2c7-40f1-8231-fc410dbffcce
127.0.0.1 - - [08/Feb/2025 01:53:54] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:57,173 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 775f803d-4a5f-4038-b38b-17cf3e7d6629
2025-02-08 01:53:57,179 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 775f803d-4a5f-4038-b38b-17cf3e7d6629
127.0.0.1 - - [08/Feb/2025 01:53:57] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:53:59,468 - LoudVA - WARNING - [LoudServer] - Timeout reached for request fd73a7f1-21c7-42a1-b68a-06c880339af3
2025-02-08 01:53:59,471 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: fd73a7f1-21c7-42a1-b68a-06c880339af3
127.0.0.1 - - [08/Feb/2025 01:53:59] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:54:02,872 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 0b588c29-cd40-47d3-a8b3-e0f47ca43899
2025-02-08 01:54:02,886 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 0b588c29-cd40-47d3-a8b3-e0f47ca43899
127.0.0.1 - - [08/Feb/2025 01:54:02] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:54:04,998 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 7ed0ed67-12b7-48fa-b188-1f04fa5223ae
2025-02-08 01:54:05,010 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 7ed0ed67-12b7-48fa-b188-1f04fa5223ae
127.0.0.1 - - [08/Feb/2025 01:54:05] "POST /inference HTTP/1.1" 200 -
2025-02-08 01:54:07,918 - LoudVA - WARNING - [LoudServer] - Timeout reached for request 43632b1b-c6ed-4419-8402-35a5fc71cb8f
2025-02-08 01:54:07,922 - LoudVA - INFO - [LoudServer] - Inference completed. Request ID: 43632b1b-c6ed-4419-8402-35a5fc71cb8f
127.0.0.1 - - [08/Feb/2025 01:54:07] "POST /inference HTTP/1.1" 200 -
